index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Introduction</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="intro__box">
+      <header class="intro__box__header flex-ct-sb">
+        <div class="intro__box-sm flex-ct-ct">
+          <a href="/">
+            <img
+              class="intro__box__image"
+              src="/assets/rocket.jpg"
+            />
+          </a>
+          <span class="intro__box-span">
+            <span class="lg sb n-s heading">Apollo-UI</span>
+            <span class="sm sb subheading">beta-version</span>
+          </span>
+        </div>
+        <div class="intro__box-sm-contact flex-ct-ct">
+          <a href="https://twitter.com/home">
+            <img
+              class="intro__box__image__sm"
+              src="https://img.icons8.com/color/48/000000/twitter--v1.png"
+            />
+          </a>
+          <a href="https://github.com/shrey27">
+            <img
+              class="intro__box__image__sm"
+              src="https://img.icons8.com/glyph-neue/48/000000/github.png"
+            />
+          </a>
+          <a href="https://www.linkedin.com/in/shrey27/">
+            <img
+              class="intro__box__image__sm"
+              src="https://img.icons8.com/color/48/000000/linkedin.png"
+            />
+          </a>
+        </div>
+      </header>
+      <main class="intro__box__main">
+        <p class="xl heading xs-s">
+          <span class="sb">Apollo-UI</span>: A versatile CSS framework that just
+          works.
+        </p>
+        <p class="md text--dark xs-s">
+          Apollo-UI is a one stop solutions to all your styling needs. Its
+          modular, clean and integrates smoothly with any framework and library.
+          Go through this documentaion to know more about usage and code
+          structure.
+        </p>
+        <div class="flex-ct-ct">
+          <a href="/installation" class="btn btn--dark btn--md">Docs</a>
```

Comment:
The link to installation page is broken. I think the html file is missing in the URL